src/core/kinesis-firehose-defaults.ts

```diff
@@ -0,0 +1,11 @@
+import { Compression, S3BucketProps } from '@aws-cdk/aws-kinesisfirehose-destinations-alpha';
```

Comment:
This is how defaults are handled in the `construct-hub` library, what do we think about this approach? I like how it enables us to centralise these defaults instead of having them all over the place at the moment

Reply:
I'm a fan, this will make precedence of settings (config, explicit props, defaults) a lot more readable in the stages as well.

Reply:
I wouldn't create a separate file for the defaults tbh but otherwise looks good to me